docs/ossm/ossm2-vs-ossm3.md

```diff
@@ -0,0 +1,103 @@
+
+## The OpenShift Service Mesh 3 operator
+
+OpenShift Service Mesh 3 uses an operator that is maintained upstream as the Sail Operator in the istio-ecosystem organization. This operator is smaller in scope and includes significant changes from the operator used in OpenShift Service Mesh 2 that was maintained as part of the Maistra.io project.
+
+## Observability integrations rather than addons
+
+A significant change in OpenShift Service Mesh 3 compared to OpenShift Service Mesh 2 is that the operator no longer installs and manages observability components such as Prometheus and Grafana with the service mesh control plane. It also no longer installs and manages distributed tracing components such as Tempo and OpenTelemetry (Jaeger and Elasticsearch in the past) or the Kiali dashboard. 
+
+The OpenShift Service Mesh 3 operator limits its scope to Istio related resources, with observability components supported and managed by the independent operators that make up [Red Hat OpenShift Observability](https://docs.redhat.com/en/documentation/openshift_container_platform/#Observability) such as [logging](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/logging/index), user workload [monitoring](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/index), [distributed tracing](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/distributed_tracing/index). The [Kiali dashboard](https://docs.openshift.com/service-mesh/3.0.0tp1/kiali/ossm-kiali-assembly.html) along with OpenShift Service Mesh Console will continue to be supported with the Kiali operator.
+
+This simplification greatly reduces the footprint and complexity of OpenShift Service Mesh, while providing better, production-grade support for observability through Red Hat OpenShift Observability.
+
+## The `Istio` resource replaces the `ServiceMeshControlPlane` resource
+
+While OpenShift Service Mesh 2 used a resource called `ServiceMeshControlPlane` to configure Istio, OpenShift Service Mesh 3 uses a resource called `Istio`. 
+
+The `Istio` resource contains a `spec.values` field that derives its schema from Istio’s Helm chart values. While this is a different configuration schema than `ServiceMeshControlPlane` uses, the fact that it is derived from Istio’s configuration means that configuration examples from the community Istio documentation can often be applied directly to Red Hat OpenShift Service Mesh’s `Istio` resource. The `spec.values` field in the `IstioOperator` resource (which is not part of OpenShift Service Mesh) has a similar format. The `Istio` resource provides an additional validation schema enabling the ability to explore the resource using the OpenShift CLI command `oc explain istios.spec.values`.
+
+## New resource: `IstioCNI`
```

Comment:
@luksa @dgn Can you have another look at this section on the IstioCNI resource... Based on the discussion in the team call, we are still using Multus in 3.0 rather than CNI chaining, and it may not have been the reason for the IstioCNI resource.  Originally, I had said it had to do with the cluster vs namespace scoping, but changed it after feedback... was it just a simplification to decouple the lifecycles for canary upgrades?  Can you suggest a better explanation for "Why" we now have the IstioCNI resource? 

If the reasoning is messy, we can gloss over it a bit - the aim of this page is to tell users what they need to know coming from OSSM 2, not how everything works under the hood.